compose-tests/src/androidTest/java/radiography/test/compose/ComposeUiTest.kt

```diff
@@ -498,10 +497,12 @@ class ComposeUiTest {
   @Test fun scanningHandlesLazyLists() {
     composeRule.setContent {
       Box(Modifier.testTag("parent")) {
-        LazyColumnFor(items = listOf(1, 2, 3), modifier = Modifier.testTag("list")) {
-          Box(Modifier.testTag("child:$it"))
-          if (it % 2 == 0) {
-            Box(Modifier.testTag("child:$it (even)"))
+        LazyColumn(Modifier.testTag("list")) {
+          items(listOf(1, 2, 3)) {
+            Box(Modifier.testTag("child:$it"))
+            if (it % 2 == 0) {
+              Box(Modifier.testTag("child:$it (even)"))
+            }
```

Comment:
Our code style changed, probably want to do an indentation pass over the whole codebase.

Reply:
#118